app/services/idv/steps/verify_base_step.rb

```diff
@@ -50,6 +50,7 @@ def add_proofing_costs(results)
             # transaction_id comes from ConversationId
             add_cost(:lexis_nexis_resolution, transaction_id: hash[:transaction_id])
           elsif stage == :state_id
+            # mawagner: Do we need to update this as well, or is this flow functionally obsolete?
             next if hash[:vendor_name] == 'UnsupportedJurisdiction'
             process_aamva(hash[:transaction_id])
```

Comment:
Ada folks: the main change is in VerifyInfoConcern. Do I need to duplicate the change here?

Reply:
This code is allllmost obsolete. Team Joy is actively working on turning on the feature flag for InPerson::VerifyInfoController, which uses VerifyInfoConcern. If the fix can wait another week for in person proofing, I recommend leaving verify_base_step unchanged. Might want to check in with Team Joy about that though.

Reply:
Team Joy had suggested that right now I do need to change it here as well.

This code path isn't well-tested, but since it's being removed soon I'm not going to jump through hoops to add test coverage on this one.